docs/configure.md

```diff
@@ -498,3 +498,8 @@ Sourcegraph's Kubernetes deployment [requires an Enterprise license key](https:/
 1. Create an account on or sign in to sourcegraph.com, and go to https://sourcegraph.com/subscriptions/new to obtain a license key.
 
 1. Once you have a license key, add it to your configuration by setting it [in the management console](https://docs.sourcegraph.com/admin/management_console).
+
+## Use namespace
+
+If you use a namespace make sure cluster role binding and service account for one of the services are in the same namespace 
```

Comment:
I would say something like :

If you're deploying Sourcegraph into a non-`default` namespace, refer to prometheus.readme.md#namespace for further configuration instructions.

Reply:
done